core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/JDBCUserStoreManager.java

```diff
@@ -3953,6 +3954,9 @@ public long doCountUsersWithClaims(String claimUri, String value) throws UserSto
             if (isUserNameClaim(claimUri)) {
                 prepStmt.setString(1, valueFilter);
                 prepStmt.setInt(2, tenantId);
+            } else if (ExpressionOperation.EQ.toString().toLowerCase().equals(valueFilter)) {
```

Comment:
Check whether we need ignore case

Reply:
Address it with commit - https://github.com/wso2/carbon-kernel/pull/3642/commits/824fe0381e39b6d89340388f531cccab35f657a1